src/components/Navbar.component.jsx

```diff
@@ -5,10 +5,17 @@ function Navbar() {
   return (
     <nav className="navbar-container">
       <div className="bar">
-        <h1>Bookstore CMS</h1>
-        <div className="links">
-          <Link className="link" to="/">Book</Link>
-          <Link className="link" to="categories">Categories</Link>
+        <div className="menu">
+          <h1>Bookstore CMS</h1>
+          <div className="links">
+            <Link className="link" to="/">Book</Link>
+            <Link className="link" to="categories">Categories</Link>
+          </div>
+        </div>
+        <div className="user">
+          <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" strokeWidth={1.5} stroke="currentColor" className="w-6 h-6 i-user">
+            <path strokeLinecap="round" strokeLinejoin="round" d="M17.982 18.725A7.488 7.488 0 0012 15.75a7.488 7.488 0 00-5.982 2.975m11.963 0a9 9 0 10-11.963 0m11.963 0A8.966 8.966 0 0112 21a8.966 8.966 0 01-5.982-2.275M15 9.75a3 3 0 11-6 0 3 3 0 016 0z" />
+          </svg>
```

Comment:
- [ ] Styling should be exactly the same provide on the [Zeplin](https://zeplin.io/). Please make sure to add the same color, font, and spacing. Please see the image below for reference.
- [ ] I would suggest you please use the credentials provided on your requirement 
### Your design
![his](https://user-images.githubusercontent.com/81727988/222482735-c988a732-624d-4393-8cae-57a22ce3064d.png)

### Original design
![original](https://user-images.githubusercontent.com/81727988/222482774-3b212714-cced-4208-b1e7-7aeb351b7023.png)